getSimulationSCSH.m

```diff
@@ -36,9 +36,11 @@
             case 'log'
                 tmp = arrayfun(@(x) diag(squeeze(SP.Cy(x,:,:))), 1:size(SP.Cy,1),'UniformOutput',false);
                 tmp = transpose([tmp{:}])/2;
-                my = [my; exp(SP.my + tmp)];
+                my_new = exp(SP.my + tmp);
+                my = [my; my_new];
                 tmpCy = bsxfun(@plus, repmat(tmp, 1, 1, size(SP.Cy,3)), permute(repmat(tmp, 1, 1, size(SP.Cy,3)), [1,3,2]));
-                Cy = [Cy; exp(tmpCy) .*  (exp(SP.Cy) - ones(size(SP.Cy)))];
+                Cy_new = exp(tmpCy) .*  (exp(SP.Cy) - ones(size(SP.Cy)));
+                Cy = [Cy; Cy_new];
```

Comment:
Also this is correct. Nice...
However: Could we, at some point in the code, write down the equations for this? Maybe as comment? I takes me half an hour to read up the formulae and double-check, that this is probably correct (assuming bsxfun, arrayfun, etc. are doing their job...). I think here, some comment would be cool...

Reply:
Can be found here at least as formula:
https://en.wikipedia.org/wiki/Log-normal_distribution#Multivariate_log-normal